packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.tsx

```diff
@@ -1,18 +1,59 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, Host, h, Element } from '@stencil/core';
 
 @Component({
   tag: 'instant-apps-app-guide',
   styleUrl: 'instant-apps-app-guide.scss',
   shadow: true,
 })
 export class InstantAppsAppGuide {
+  @Element()
+  el: HTMLInstantAppsAppGuideElement;
 
   render() {
     return (
       <Host>
-        <slot>It Works!</slot>
+        <calcite-panel scale="s">
+          <span slot="header-content">App Guide <calcite-icon icon="lightbulb" scale="s"></calcite-icon></span>
+          <calcite-carousel arrow-type="none">
+            <calcite-carousel-item>
+              <div>
+                {/* Begin: App Guide Page */}
+                <b class="content-heading">Page One</b>
+                <p>Lorem ipsum dolor sit amet, amet ullamco Lorem enim nulla est. </p>
```

Comment:
Let's move the hardcoded lorem ipsum strings outside of the component and pass it into the component and render out the passed in data via jsx

Reply:
Done!